README.md

```diff
@@ -217,3 +230,14 @@ This approach has a few major advantages:
 ## License
 
 [Apache 2.0](LICENSE)
+
+[circleci]:
+  https://circleci.com/gh/nasa/cumulus-message-adapter-python.svg?style=svg
+[pypi version]:
+  https://badge.fury.io/py/cumulus-message-adapter-python.svg
+[Cumulus Documentation]:
+  https://cumulus-nasa.github.io/
```

Comment:
This isn't your error, but this link is wrong. It should be `https://nasa.github.io/cumulus/`

Reply:
Thanks. Fixed.